README.md

```diff
@@ -11,7 +11,7 @@ or feedback.
 
 ## What are the Good Labs?
 
-Transparency, including open communication, incentives, and funding are essential
+Transparency, including [open communication](https://good-labs.github.io/docs/projects/slopi-communication/), incentives, and funding are essential
```

Comment:
any reason to not use site.baseurl here?

Also, is there a sidebar name that could be shorter (abbreviated?) For example, instead of spelling SLOPI out, why not just put "Open Communication" and then the user is directed to read about SLOPI. It looks much cleaner.

![image](https://user-images.githubusercontent.com/814322/58370833-25482980-7ed9-11e9-92d8-246cf7190a9e.png)

Reply:
> any reason to not use site.baseurl here?

I don't believe it's possible to use site.baseurl in a README. I just tried at https://github.com/pdurbin/good-labs.github.io/blob/a6428b08d7615a3fb1fbdb6713ce4f9d9f224426/README.md

Reply:
> a sidebar name that could be shorter

Hmm, it is pretty long. How about "SLOPI Communication"?

Reply:
That sounds good.

Reply:
Oh, re the README, if it's outside of the docs folder you are correct! I overlooked the location.

Reply:
@vsoch ok, shortened in 0080079